app/views/dashboard/index.html.erb

```diff
@@ -1,13 +1,25 @@
 <%= javascript 'dashboard' %>
 <% title _('Overview') %>
 <%= title_actions dashboard_actions %>
+
 <div class="row">
   <div id='dashboard' class="gridster col-md-12">
+  <% if User.current.widgets.empty? %>
+    <%= _("No widgets defined. Why not add the defaults from the menu?") %>
```

Comment:
This could be a centered, big button `Get default dashboard widgets`. It is a big cryptic now as the user has to think where is the button to add the default widgets (I didn't find it myself)